### PR TITLE
Added optional attributes for passing suggesstion and nope text

### PR DESCRIPTION
**Pull request by tomislijepcevic**

### CODE REVIEW

**Comment by simison** [comment]
Aawsom. Will test right away.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ angular.module('myApp', ['angular-mailcheck']);
 
 ```html
 <input type="email" mailcheck>
+
+<!-- You can add suggestion and nope text -->
+<input type="email" mailcheck mailcheck-suggestion-text="suggestionText" mailcheck-nope-text="nopeText">
+
+<!-- When using angular-translate -->
+<input type="email" mailcheck mailcheck-suggestion-text="'EMAIL_SUGGESTION_OFFER' | translate" mailcheck-nope-text="'EMAIL_SUGGESTION_PASS' | translate">
 ```
```
Comment (simison) [inline README.md:48]: Thanks. Documentation could be something like:

---

``` html
<input type="email" mailcheck>
```

You can add suggestion and nope texts as strings:

``` html
<input type="email" mailcheck mailcheck-suggestion-text="'Did you mean:'" mailcheck-nope-text="'Nope'">
```

...or as scope variables: 

``` html
<input type="email" mailcheck mailcheck-suggestion-text="suggestionText" mailcheck-nope-text="nopeText">
```

...or use [Angular-Translate](https://angular-translate.github.io/):

``` html
<input type="email" mailcheck mailcheck-suggestion-text="'EMAIL_SUGGESTION_OFFER' | translate" mailcheck-nope-text="'EMAIL_SUGGESTION_PASS' | translate">
```

---

What do you think?

Comment (tomislijepcevic) [inline README.md:48]: I agree. Will change right away.

**Comment by simison** [comment]
Cheers! 🍻

**Comment by simison** [comment]
You can now get this via tag 0.2.0

https://github.com/simison/angular-mailcheck/releases

**Pull request merged**